docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,68 @@
+# Release 450 (7 Jun 2024)
+
+## General
+
+* Add support for the `optimizer.push-filter-into-values-max-row-count`
+  configuration property, which can define a number of rows to be pushed from a
+  filter into values. ({issue}`22105`)
+* Improve performance of the {func}`first_value` and {func}`last_value`
+  functions. ({issue}`22092`)
+* Improve performance for large clusters under heavy workloads. ({issue}`22039`)
+* Improve performance of queries with simple predicates. This optimization can
```

Comment:
is "simple predicates" specific enough? I kinda think yes and people can look up more if needed. Also .. is this documented? And "experimental" .. do we need to mention that besides the name of the property?